docs/reference/query-dsl/function-score-query.asciidoc

```diff
@@ -415,6 +415,17 @@ Example:
     }
 --------------------------------------------------
 
+[[function-query]]
+==== Query function
+
+If the score for a function again should be computed by a different query, use the `query_function`:
+
+[source,js]
+--------------------------------------------------
+"query_score": {
+    ARBITRARY QUERY
+}
+--------------------------------------------------
```

Comment:
I think it is important to mention that this produces a score of 0 if the query does not match.